doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -276,34 +275,66 @@ At a high level, you can compare these concepts as follows:
      - n/a
      - :meth:`ray.data.Dataset.iter_torch_batches`
 
+Why using Ray Data?
+~~~~~~~~~~~~~~~~~~~
+
+The framework's data utilities work well with small datasets requiring light preprocessing. 
+However, they can become performance bottlenecks when handling large-scale datasets with complex preprocessing logic. 
+Ray Data is designed to address these challenges, providing efficient large-scale data ingestion. 
+
+Specifically, you can benefit from the following features of Ray Data:
+
+**Streaming execution**:
+
+- The preprocessing pipeline will be executed lazily and stream the data batches into training workers.
+- Training can start immediately without significant up-front preprocessing time.
+
+**Automatic data sharding**: 
+
+- The dataset will be automatically sharded across all training workers. 
 
-For more details, see the following sections for each framework.
+**Leverage additional resources for preprocessing**
+
+- Ray Data can utilize all resources in the Ray cluster for preprocessing, not just those on your training nodes. 
```

Comment:
This is good content that I think everyone should read, regardless of whether or not they are starting with PyTorch data. Do you think we could bring this higher up in the guide (e.g. even in the introduction), and then reference it from here?

Reply:
OK. Sounds good to me.